### PR TITLE
Add Enjoy2.app latest

### PR DESCRIPTION
**Pull request by kennethzfeng**
Hi,

I added [fyhuang/enjoy2](https://github.com/fyhuang/enjoy2), a joystick mapping app.  Please review.

The author provided an URL to a binary version of the app (v1.2.1).  However, since the version is not in the URL, I used the `:latest` version.

### CODE REVIEW

File: Casks/enjoy.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'enjoy' do
+  version ':latest'
+  sha256 '95f37aae546392697a55c3e72add68dadff67663ceba0b9f86815c867cc9e85d'
+
+  url 'http://nongraphical.s3-website-us-east-1.amazonaws.com/releases/Enjoy2.zip'
+  name 'Enjoy2'
+  homepage 'http://nongraphical.com/'
```
Comment (vitorgalvao) [inline Casks/enjoy.rb]: Correct homepage is https://github.com/fyhuang/enjoy2/.

**Comment by kennethzfeng** [comment]
Hi @Ngrd and @vitorgalvao,

Thanks for the helpful comments.  I updated the ```enjoy.rb``` per your suggestion.  Do we need the vendor/author's permission to add this formula to homebrew-cask?

**Comment by vitorgalvao** [comment]
> Do we need the vendor/author's permission to add this formula to homebrew-cask?

No, we’re simply linking to it, nothing more. You can tell them, if you so wish, and they can ask us to remove it.

Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by kennethzfeng** [comment]
@vitorgalvao,

squashed and placed it on the latest HEAD on master.

**Comment by Ngrd** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 4377b3118d0e980ae0cc481b7647fd24e8c3518a. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**